R/vaccine_parameters.R

```diff
@@ -1,35 +1,106 @@
-#' @title Parameterise an RTS,S strategy
+#' @title Parameterise an RTS,S epi strategy
+#'
+#' @description distribute RTS,S vaccine when an individual becomes a certain
+#' age. Efficacy will take effect after the last dose
+#'
+#' @param parameters a list of parameters to modify
+#' @param start timestep to turn on epi vaccination
+#' @param end timestep to turn off epi vaccination
+#' @param coverage the coverage for the starter doses
+#' @param age for the target population, (in timesteps)
+#' @param min_wait the minimum acceptable time since the last vaccination (in
+#' timesteps); This includes seasonal boosters and vaccinations between EPI and mass
+#' strategies
```

Comment:
```suggestion
#' @param min_wait the minimum acceptable time since the last vaccination (in
#' timesteps); When seasonal_boosters = TRUE, this represents the minimum time
#' between an individual receiving the final dose and the first booster. When using
#' both set_mass_rtss and set_rtss_epi, this represents the minimum time between
#' an individual being vaccinated under one scheme and vaccinated under another.
```